### ISSUE TITLE
Conflict with Jetpack site stats 

### ISSUE THREAD

**Issue by chaselivingston** [ahmedkaludi/accelerated-mobile-pages#1308]
We had a report that version 0.9.65 of Accelerated Mobile Pages breaks the Dashboard stats view for Jetpack's site stats. These detailed stats are available at `/wp-admin/admin.php?page=stats` but just load endlessly with version 0.9.65. Downgrading to 0.9.64 solves the problem.

**Comment by ahmedkaludi** [comment]
Can you share a screenshot or an image? 

We will definitely test and fix this in the next update which will be released on Saturday.

**Comment by ahmedkaludi** [comment]
Here is another reference URL: https://wordpress.org/support/topic/no-more-jetpack-statistic-with-0-9-64/#post-9617577

**Comment by chaselivingston** [comment]
Here's a screenshot of the loading spinner for Jetpack Stats from the Dashboard when v0.9.65.1 is installed and activated.

![screen shot on 2017-10-25 at 08 3a30 3a40](https://user-images.githubusercontent.com/831377/31998772-f6140bfc-b95e-11e7-89c7-7d5341eb1afd.png)

**Comment by chaselivingston** [comment]
Here's a possibly related error that one of our users found in their error logs:

```
(index):520 WebSocket connection to 'wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data' failed: Error during WebSocket handshake: Unexpected response code: 403
```

**Comment by MohammedKaludi** [comment]
Hey@chaselivingston,

We have fixed the issue in #1299 and #1303 and released an update last night.

Please update the plugin and clear your browser's cache and it should fix the issue.
Also, we got the confirmation that issue has been fixed in the latest update from WP forum 
https://wordpress.org/support/topic/no-more-jetpack-statistic-with-0-9-64/page/2/

Thank you very much for reporting the issue,
Mohammed Kaludi
